guides/common/modules/proc_upgrading-a-connected-project-server.adoc

```diff
@@ -76,6 +77,41 @@ ifdef::katello[]
 # foreman-rake katello:upgrade_check
 ----
 endif::[]
+ifdef::foreman-deb[]
+. Update repositories in `/etc/apt/sources.list.d/foreman.list` to consume content for {Project} {ProjectVersion}:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+deb http://deb.theforeman.org/ _My_Distribution_Code_Name_ {ProjectVersion}
+deb http://deb.theforeman.org/ plugins {ProjectVersion}
+----
++
+// List of supported OS for foreman-deb has to match "guides/common/modules/proc_configuring-repositories-foreman-deb.adoc"
+Replace _My_Distribution_Code_Name_ with the proper distribution code name based on the operating system of your {ProjectServer}:
++
+* `bullseye` for Debian 11
+* `jammy` for Ubuntu 22.04
+* `focal` for Ubuntu 20.04
+. Stop all {Project} services:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# {foreman-maintain} service stop
+----
+. Update to {Project} {ProjectVersion}:
++
+[options="nowrap" subs="attributes"]
+----
+# apt-get update
+# apt-get upgrade
+----
```

Comment:
The flow here is a bit weird:
* change repo config (apt)
* stop services (foreman)
* update packages (apt)

While technically correct (changing repo config doesn't further alter the system), I'd still go with:
* stop services (foreman)
* change repo config (apt)
* update packages (apt)

As it creates less context switches.

Reply:
Makes sense, fixed.